### ISSUE TITLE
CAL-AMP17 Provide a valid label for form fields

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13530]
**monday.com sync:** #10821961869

### Violation: 
 Provide a valid label for form fields 

### WCAG Reference: 
- [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name) 

### Severity: 
 Critical 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP17 Provide a valid label for form fields 
- Module: M3: Form validation_Calcite Feedback Survey 
- Finding ID: [CAL-AMP17](https://esri.hub.essentia11y.com/short-link/5701t9UayFObhnfk) 

**Issue**
There are form controls with visual labels that are not programmatically associated with the control. Examples include:
- How many components are you familiar with?
- What industry are you in?
- How usable is the Calcite Design System?
- When did you hear about Calcite components?
- How many stars would you give Calcite components?.

**User Impact**
Screen reader users will have difficulty determining the purpose of these controls. Speech input users will have difficulty navigating to them.

**Code Reference** 
 ```
<div aria-orientation="horizontal" role="slider" tabindex="0" aria-valuemax="100" aria-valuemin="0" aria-valuenow="0" class=" thumb thumb--value " data-value-prop="value" style="right:100%;">(...)</div>

```

**Steps to Reproduce**
1. Locate the form field.
2. Inspect it with Chrome DevTools.
3. In the Accessibility tab, expand the Computed Properties section.
4. Review the value for "Name".
 
**Recommendation**
Provide a valid label for form fields. The best way to do this is to use a ```<label>``` element with a for attribute set to the ID of the field it labels. For custom controls, use aria-labelledby on the control set to the ID of the labelling text.

**Compliant Code Example**
```
<!--Example using label element--> 
<label for="amount ">Amount:</label> <input id="Amount" type="text" name="amount" size="30"> 
---
<!--Example label has no "for" attribute but does wrap the input element)--> 
<label>Amount: <input id="amount " type="text" name="amount " size="30"></label> 
---
<!--Example using aria-labelledby--> 
<label id="lblAmount ">Amount:</label> <input aria-labelledby="lblAmount" id="Amount" type="text" name="amount" size="30"> 
---
<!-- Example using aria-label attribute - visual label exists but is not associated--> 
<span>Amount:</span> 
<input aria-label="Amount:" type="text" id="amount" size="30"> 
---
<!--Example using title attribute - visual label exists but is not associated--> 
<span>Amount:</span> 
<input title="Amount:" type="text" id="amount" size="30">
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP17.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/5701t9UayFObhnfk) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.

**Comment by geospatialem** [comment]
Duplicate of #13529
